Makefile

```diff
@@ -16,6 +16,8 @@ BINARIES=$(addprefix bin/,$(COMMANDS))
 
 GO_LDFLAGS=-ldflags "-X `go list ./version`.Version=$(VERSION)"
 
+DOCKER_BUILDTAGS+="experimental"
```

Comment:
No, any API call will simply fail if engine doesnt support it. For nicer UX, we could populate the daemon's capabilities in `docker info`. This issue probably applies to all new endpoints like `checkpoint`, `system`, etc